config.example.toml

```diff
@@ -180,7 +180,7 @@ EnableCSRFProtection = false
     [Storage.Minio]
         # Endpoint for Minio storage
         # Env override: ATHENS_MINIO_ENDPOINT
-        Endpoint = "minio.example.com"
+        Endpoint = "127.0.0.1:9000"
```

Comment:
9001?

Reply:
Since it's an example config i've put the default minio port here similar to the mongo part.
I've only changed it in our docker-compose file and the test config so it doesn't conflict with other services on a dev machine.

Reply:
yes i understand intentions.
but latest change here: https://github.com/gomods/athens/pull/678/files
uses example as a default for proxy docker

Reply:
@michalpristas I'll revert it to the old value (minio.example.com) so it stays as generic as possible. As per slack discussion - we should probably use the config.test.toml in our docker img.